docs/events.md

```diff
@@ -428,6 +501,54 @@ var GrantParentView = Marionette.View.extend({
 
 [Live example](https://jsfiddle.net/marionettejs/8eq7vca5/)
 
+#### Using `CollectionView`'s `childViewTriggers`
+
+```javascript
+var Mn = require('backbone.marionette');
+
+// The child view fires a custom event, `show:message`
+var ChildView = Mn.View.extend({
+
+  // Events hash defines local event handlers that in turn may call `triggerMethod`.
+  events: {
+    'click .button': 'onClickButton'
```

Comment:
I just moved this from elsewhere.. although I typically follow naming my function names after the event in this way.  Then inside those functions it will procedurally call whatever needs to happen even if it is only `showMessages`.  Typically I follow this because when in the future the button needs to show message and then do something else do you rename the function?  This is also the functionality behind `triggerMethod` so I find it consistent.  Right now, I suppose, my naming structure can be dangerous because of auto-proxying, but hopefully that can be disabled soon.